build.bat

```diff
@@ -63,7 +63,8 @@ set "custom_idl_file=%custom_type_folder%\custom.idl"
 
 @REM # Variables for FlatData
 set "flatdata_size=10485760" @REM # 10MB
-set flatdata_ddsgen_version=300
+@REM #3.0.0 -> 3 We just need the Mayor value of the version.
```

Comment:
Is is marking me this character as conflictive... Maybe use just the `--`

Reply:
Done